packages/lesmis/package.json

```diff
@@ -15,16 +15,10 @@
     "preview": "vite preview",
     "lint": "yarn package:lint",
     "lint:fix": "yarn lint --fix",
-    "start-fullstack": "npm-run-all -c -l -p start-servers start-dev",
-    "start-central-server": "yarn workspace @tupaia/central-server start-dev",
-    "start-entity-server": "yarn workspace @tupaia/entity-server start-dev",
-    "start-report-server": "yarn workspace @tupaia/report-server start-dev",
-    "start-web-config-server": "yarn workspace @tupaia/web-config-server start-dev",
-    "start-lesmis-server": "yarn workspace @tupaia/lesmis-server start-dev",
+    "start-fullstack": "yarn package:start-stack lesmis",
```

Comment:
I kind of like the idea of having this be `start-stack` everywhere, ie:
```
yarn start-stack (starts tupaia-web by default)
yarn package:start-stack <package> (starts stack for package)
cd packages/<package> && yarn start-stack (starts stack for package)
```
Saves having to remember multiple commands, and might appreciate saving a few keystrokes from `start-fullstack`

Thoughts?

Reply:
Yeah I like that better too. I think I left it as fullstack because I didn't want to step on the toes of whoever came up with that name originally :) But easy enough to change back if someone is married to it. Will update.

Reply:
Updated